ui/lib/kv/addon/components/kv-delete-modal.js

```diff
@@ -20,7 +20,7 @@ import { assert } from '@ember/debug';
  *
  * @param {string} mode - delete, delete-metadata, or destroy.
  * @param {object} secret - The kv/data model.
- * @param {object} [metadata] - The kv/metadata model. It is only required when mode is "delete" or "metadata-delete".
+ * @param {object} [metadata] - The kv/metadata model. It is only required when mode is "delete".
```

Comment:
Looking at the code, this didn't seem true. It appeared that the metadata model was only required for `delete` which renders different delete options, please check me on this!

Reply:
I say this with little brain power at the moment. I believe the metadata-delete from the metadata model is used on the list view or the metadata tab view. So I believe you're correct. Probably a carry over from when we went back and forth if we were adding permanently delete to the modal, which we opted not to do.